tutorials/navigation/navigation_using_navigationmeshes.rst

```diff
@@ -17,254 +19,320 @@ Navigation meshes are the only things considered when doing pathfinding, e.g. vi
-        var region_rid: RID = get_region_rid()
-        NavigationServer3D.region_set_navigation_mesh(region_rid, navigation_mesh)
+        When a NavigationRegion3D node is selected in the Editor bake options appear in the top bar of the Editor.
 
-.. note::
+        .. image:: img/nav_mesh_bake_toolbar.webp
 
-    Baking a NavigationMesh at runtime is a costly operation.
-    A complex navigation mesh takes some time to bake and if done on the main thread can freeze a game.
-    (Re)baking a large navigation mesh is preferably done in a separate thread.
+        In order for the region to work a :ref:`NavigationMesh<class_NavigationMesh>` resource needs to be added.
 
-.. warning::
+        The properties to parse and bake a navigation mesh are then part of the used resource and can be found in the resource Inspector.
 
-    Property values on a NavigationMesh resource like ``cell_size`` need
-    to match the actual mesh data stored inside in order to merge
-    different navigation meshes without issues.
+        .. image:: img/nav_region3d_baking_01.webp
 
-NavigationRegion2D and NavigationRegion3D both use meshes to mark traversable areas, only the tools to create them are different.
+        The result of the source geometry parsing can be influenced with the following properties.
 
-For 2D, NavigationPolygon resources are used to draw outline points in the editor. From these outline points the NavigationServer2D creates a mesh to upload navigation data to the NavigationServer.
+        - The ``parsed_geometry_type`` that filters if visual objects or physics objects or both should be parsed from the :ref:`SceneTree<class_SceneTree>`. For more detail what objects are parsed and how see the section about parsing source geometry further below.
+        - The ``collision_mask`` filters which physics collision objects are included when the 'parsed_geometry_type' includes static colliders.
+        - The ``source_geometry_mode`` that defines on which node(s) to start the parsing and how to traverse the :ref:`SceneTree<class_SceneTree>`.
+        - The ``source_geometry_group_name`` is used when only a certain node group should be parsed. Depends on the selected 'source_geometry_mode'.
 
-For 3D, NavigationMesh resources are used. Instead of providing draw tools the 3D variant
-provides an extensive amount of parameters to bake a navigation mesh directly from 3D source geometry.
+        With the source geometry added the result of the baking can be influenced with the following properties.
 
-.. note::
+        - The ``cell_size`` and ``cell_height`` that sets the rasterization voxel grid cells and should match the navigation map size.
+        - The ``agent_radius`` that shrinks the baked navigation mesh to have enough margin for the agent (collision) size.
+        - The ``agent_height`` that excludes places from the navigation mesh where the agent can not fit in.
```

Comment:
```suggestion
        - The ``agent_height`` excludes areas from the navigation mesh where the agent is too tall to fit in.
```
Specificity IMO, even though it could be considered obvious